projects/online/online/utils/gdb.py

```diff
@@ -178,7 +179,14 @@ def submit_low_latency_pe(
         posterior_samples = posterior_df.to_records(index=False)
         with h5py.File(filename, "w") as f:
             f.create_dataset("posterior_samples", data=posterior_samples)
-        self.write_log(graceid, "posterior", filename=filename, tag_name="pe")
+
+        _, has_ns, _, _ = em_bright.source_classification_pe(
+            filename, num_eos_draws=10
+        )
+        if has_ns == 0:
+            self.write_log(
+                graceid, "posterior", filename=filename, tag_name="pe"
+            )
```

Comment:
Let's leave a log message on _our_ logger with the graceid and the HasNS value, so that we can refer to it later. Also equalities like this probably work i.e. agnostic of float vs int, but to be safe, let's make this a `if has_ns > 0: #write to our log...else: # upload to gracedb`